include/oc_ri.h

```diff
@@ -397,6 +420,16 @@ void oc_ri_remove_timed_event_callback(void *cb_data,
  */
 int oc_status_code(oc_status_t key);
 
+
+/**
+ * @brief convert the status code to string
+ *
+ * @param[in] key key the application level key of the code
+ * @return char* CoAP status code string
+ */
+const char *oc_status_to_str(oc_status_t key);
```

Comment:
Add a `OC_API` annotation since it is public:

```
OC_API
const char *oc_status_to_str(oc_status_t key);
```

Reply:
Fixed, 
do you think that `OC_API` should also be added to the new function such as `oc_set_string()` in `oc_helpers.*` ?

Reply:
Yeah, it should have it, so played add it. 

I haven't got  around yet to updating old code, but we should add it when we create new functions, so we don't increase the amount of work necessary to make .DLLs on Windows work.

Reply:
done